src/docfx/lib/YamlUtility.cs

```diff
@@ -100,7 +100,7 @@ public static (List<Error> erros, JToken token) Deserialize(TextReader reader)
             }
             catch (YamlException ex)
             {
-                errors.Add(Errors.YamlSyntaxError(ex));
+                errors.Add(Errors.YamlSyntaxError(ex.InnerException));
```

Comment:
you want to test other exceptions, they may not have an inner exception

Reply:
true, still use exception and assert contain in the test